integration-testing/tests/users/set-dating-status.test.js

```diff
@@ -0,0 +1,144 @@
+const fs = require('fs')
+const path = require('path')
+const uuidv4 = require('uuid/v4')
+
+const cognito = require('../../utils/cognito')
+const {mutations, queries} = require('../../schema')
+
+let anonClient
+const grantData = fs.readFileSync(path.join(__dirname, '..', '..', 'fixtures', 'grant.jpg'))
+const grantDataB64 = new Buffer.from(grantData).toString('base64')
+const loginCache = new cognito.AppSyncLoginCache()
+jest.retryTimes(1)
+
+beforeAll(async () => {
+  loginCache.addCleanLogin(await cognito.getAppSyncLogin())
+  loginCache.addCleanLogin(await cognito.getAppSyncLogin())
+})
+beforeEach(async () => await loginCache.clean())
+afterAll(async () => await loginCache.reset())
+afterEach(async () => {
+  if (anonClient) await anonClient.mutate({mutation: mutations.deleteUser})
+  anonClient = null
+})
+
+test('Validate user dating status permission', async () => {
+  const {client: ourClient, userId} = await loginCache.getCleanLogin()
+
+  // Validate user dating status permission
+  await expect(
+    ourClient.mutate({
+      mutation: mutations.setUserDatingStatus,
+      variables: {
+        status: 'ENABLED',
+      },
+    }),
+  ).rejects.toThrow(/ClientError: `fullName` is required field/)
+
+  // Set fullName
+  await ourClient.mutate({mutation: mutations.setUserDetails, variables: {fullName: 'Hunter S'}})
+  await expect(
+    ourClient.mutate({
+      mutation: mutations.setUserDatingStatus,
+      variables: {
+        status: 'ENABLED',
+      },
+    }),
```

Comment:
```suggestion
    ourClient.mutate({mutation: mutations.setUserDatingStatus, variables: {status: 'ENABLED'}}),
```

This applies to all calls to `mutations.setUserDatingStatus` in this file